BoxContentSDK/BoxContentSDK/OAuth2/BOXAppUserSession.m

```diff
@@ -24,25 +24,9 @@ - (void)performAuthorizationCodeGrantWithReceivedURL:(NSURL *)URL withCompletion
         weakSelf.accessToken = accessToken;
         weakSelf.accessTokenExpiration = accessTokenExpiration;
         
-        BOXUserRequest *userRequest = [[BOXUserRequest alloc] init];
-        [weakSelf prepareRequest:userRequest];
-        
-        [userRequest performRequestWithCompletion:^(BOXUser *user, NSError *error) {
```

Comment:
Removing this is not a backwards-compatible change for the non-app-user case, I believe. How do we at least get our automatic keychain storage to work for the normal Box user case?

Reply:
@jlawton  so actually it is (I believe) as that session is only used for app user sessions.  The other sessions are un-modified.  Anyway, I just checked in a big change where I realized I can eliminate the wrapper and just move this functionality into BOXContentClient with a new initializer class method.  Have a look and let me know what you think.  thanks!

Reply:
@jlawton  I don't believe I've changed anything with the existing user models

Reply:
Ah. Great. I'll take another look.